src/Carcassonne/Game/Game.cpp

```diff
@@ -518,4 +523,249 @@ std::pair<Direction, int> Game::move_score(Player player, TileType tile_type, Ti
    return std::make_pair(best_dir, total_score);
 }
 
+template <typename E>
+constexpr inline typename std::underlying_type<E>::type to_underlying(E e) noexcept {
```

Comment:
Jak dajesz constexpr to dostajesz inilne za darmo i nie jestem pewien czy to jest dobry pomysł